controllers/keda/hpa.go

```diff
@@ -63,7 +63,7 @@ func (r *ScaledObjectReconciler) createAndDeployNewHPA(ctx context.Context, logg
 
 	err = kedastatus.UpdateScaledObjectStatus(ctx, r.Client, logger, scaledObject, status)
 	if err != nil {
-		logger.Error(err, "Error updating scaledObject status with used hpaName")
+		logger.Error(err, "Failed to update scaledObject status with used hpaName")
```

Comment:
I know it is already merged, but go error messages should start with lowercase, we should slowly reflect that in the codebase when we are doing a new changes.

Reply:
ok